src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -542,7 +542,9 @@ public sealed partial class Interaction
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public static string? GetSetting(string AppName, string Section, string Key, string? Default = "") { throw null; }
         public static object? IIf(bool Expression, object? TruePart, object? FalsePart) { throw null; }
+        [System.Obsolete(System.Obsoletions.VisualBasicInputBoxMessage, error: false, DiagnosticId = System.Obsoletions.VisualBasicInputBoxDiagId, UrlFormat = System.Obsoletions.SharedUrlFormat)]
         public static string InputBox(string Prompt, string Title = "", string DefaultResponse = "", int XPos = -1, int YPos = -1) { throw null; }
+        [System.Obsolete(System.Obsoletions.VisualBasicMsgBoxMessage, error: false, DiagnosticId = System.Obsoletions.VisualBasicMsgBoxDiagId, UrlFormat = System.Obsoletions.SharedUrlFormat)]
         public static MsgBoxResult MsgBox(object Prompt, MsgBoxStyle Buttons = MsgBoxStyle.ApplicationModal, object? Title = null) { throw null; }
```

Comment:
Do I need to apply the same to the actual implementations in here?
https://github.com/dotnet/runtime/blob/760a76bf4dc7d485f86c02dfe55713c75ee0121c/src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Interaction.vb#L129-L135

If yes, we can't link Obsoletions.cs in that project, and we'll need to create a VB version of it...

Reply:
> Do I need to apply the same to the actual implementations in here?

Yes.

You do not need to create a full blown VB version of Obsoletions.cs for this. It should be good enough to just add the attributes here with inlined strings.